src/main/java/com/sourcegraph/webhook/WebhookListener.java

```diff
@@ -35,8 +38,30 @@
     }
 
     @EventListener
-    public void onPullRequestEvent(PullRequestActivityEvent event) {
-        String key = "pr:activity:status";
+    public void onPullRequestEvent(PullRequestEvent event) {
+        String key = "pr:event";
+
+        // When review status changes, two events are fired in most cases
```

Comment:
What does "in most cases" mean? It's a bit unnerving to read "most" without further qualification in code documentation 😄